teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -548,6 +549,28 @@ interface Change {
     val date: Date
 }
 
+interface ChangeFiles {
+    val count: Int
+    val files: List<ChangeFile>
+}
+
+interface ChangeFile {
+    val fileRevisionBeforeChange: String?
+    val fileRevisionAfterChange: String?
+    val changeType: ChangeType
+    val filePath: String?
+    val relativeFilePath: String?
+}
+
+enum class ChangeType {
+    EDITED,
+    ADDED,
+    REMOVED,
+    COPIED,
```

Comment:
The first 3 types are self-describing, but `COPIED` looks strange. What does it mean? Git tracks file movement, but it doesn't track copying as far as I know.